### ISSUE TITLE
implement broadcast variables

### ISSUE THREAD

**Issue by timodonnell** [hammerlab/kubeface#19]
Some jobs have large amounts of constant data that needs to be available to all tasks. One way to implement this would be to wrap such values with a `Broadcast` class that when pickled saves the wrapped value to bucket storage (perhaps named according to hash of value) and when depickled reads the value

**Comment by timodonnell** [comment]
Sketch of how this could be used

```py
client = kubeface.Client(…)
big_training_data = pandas.read_csv(“path_to_training.csv”)
broadcasted_training_data = client.broadcast(big_training_data)

def train_a_model(data, model):
  unwrapped_data = data.value  # data is a kubeface.Broadcast
  return model.fit(unwrapped_data)

models = grid_of_models_to_try()
results = client.map(functools.partial(train_a_model, broadcasted_training_data), models)

**Comment by timodonnell** [comment]
I think you were right to close this ( @julia326 implemented broadcast vars in #28 )
